assets/charts/components/rook/templates/prometheus-ceph-v14-rules-for-prometheus-operator-0.43.2.yaml

```diff
@@ -0,0 +1,262 @@
+# This resource is created to scrap rook metrics once Rook component is updated.
```

Comment:
We should mention in the commit message when is the right time to remove it? Like after release of `0.6.0` (with an assumption that this PR will go as a part of 0.6.

Reply:
In fact creating an issue is more beneficial in this case.

Reply:
Created #1212